pravega-sensor-collector/src/main/java/io/pravega/sensor/collector/util/ObjectSerializer.java

```diff
@@ -0,0 +1,46 @@
+package io.pravega.sensor.collector.util;
+
+import io.pravega.client.stream.Serializer;
+
+import java.io.*;
+import java.nio.ByteBuffer;
+
+public class ObjectSerializer implements Serializer<Object>, Serializable {
+    /**
+     * Serializes the given event.
+     *
+     * @param value The event to be serialized.
+     * @return The serialized form of the event.
+     * NOTE: buffers returned should not exceed {@link #MAX_EVENT_SIZE}.
+     */
+    @Override
+    public ByteBuffer serialize(Object value) {
+        try(ByteArrayOutputStream b = new ByteArrayOutputStream()){
+            try(ObjectOutputStream o = new ObjectOutputStream(b)){
+                o.writeObject(value);
```

Comment:
It is unclear how this is serializing generic objects. Is it using Java serialization? If so, we want to avoid that because that makes it nearly impossible to read from non-Java languages. We usually want to use JSON serialization.

Reply:
The writeObject use serialization written by ObjectStream class. To avoid cross language problems as mentioned , Serializer will be changed to existing byte array serializer used by Accelerometer driver and relevant changes will be made.